packages/ui-kit/index.ts

```diff
@@ -143,6 +149,10 @@ export {
     Dialog,
     TextOverlay,
     Modal,
+    ModalHeader,
+    ModalBody,
+    ModalTitle,
+    ModalFooter,
```

Comment:
looks like we forgot to export these

Reply:
That's because Modals were placed here  `packages/ui-kit/src/components/modal`
<img width="411" alt="Screenshot 2024-08-08 at 21 33 46" src="https://github.com/user-attachments/assets/9ceef11a-e107-4d53-8a2e-52ce49a6c9ca">